coding/using_boost.rst

```diff
@@ -4,24 +4,38 @@ Using Boost
 
 .. _cpp_using_boost:
 
-Certain Boost libraries are recommended; they should be used whenever applicable in preference to any other method of accomplishing the same effect. In general, any library that is tagged with *"Standard .... {something}"* in the `Boost library listing <http://www.boost.org/doc/libs>`_ falls into this category, unless the feature is already supported by the currently-mandated LSST ``g++`` version.
-Among others, this category includes:
+A Boost library shall be used in in preference to any other method of accomplishing the same effect only if:
```

Comment:
Is the logic here correct? As I read what you've written, if the three criteria below are satisfied you *should* use Boost, even if other libraries which might do a better job (but which aren't in the standard) are available. I think you probably mean "may" rather than "shall".